packages/app-bridge/src/react/useDocumentPages.ts

```diff
@@ -12,17 +12,38 @@ export type DocumentPageEvent = {
     documentPage: DocumentPage | { id: number };
 };
 
-const sortDocumentPages = (a: DocumentPage, b: DocumentPage) => (a.sort && b.sort ? a.sort - b.sort : 0);
-
-export const useDocumentPages = (appBridge: AppBridgeBlock | AppBridgeTheme, documentId: number) => {
+export const useDocumentPages = (
+    appBridge: AppBridgeBlock | AppBridgeTheme,
+    documentId?: number,
+    categoryId?: number,
+    allPages?: boolean,
+) => {
```

Comment:
The whole hook now feels hacky, please team up with someone else and try to get a cleaner hook interface

Reply:
Yeah I know, that's why I left it in draft, this was more of a quick implementation to test things out in clarify.